draft-ietf-quic-recovery.md

```diff
@@ -118,8 +118,8 @@ ACK-only:
 In-flight:
 
 : Packets are considered in-flight when they have been sent
-  and neither acknowledged nor declared lost, and they are not
-  ACK-only.
+  and neither acknowledged nor declared lost nor abandoned due
+  to keys being discarded, and they are not ACK-only.
```

Comment:
suggestion: "Packets are considered in-flight when they have been sent and are not ACK-only, and they are not acknowledged, declared lost, or abandoned along with old keys."